app/views/idv/doc_auth/document_capture.html.erb

```diff
@@ -10,7 +10,7 @@
       content="<%= Figaro.env.acuant_sdk_initialization_creds %>"
     >
   <% end %>
-  <div id="document-capture-form"></div>
+  <div id="document-capture-form"<%= ' data-liveness' if liveness_checking_enabled? %>></div>
```

Comment:
WDYT of the rails tag helpers for this so we don't have to mangle strings? A `nil` value from (`prescence` converts all falsy values to nil) will be omitted in the rendered HTML

```erb
<%= tag.div id: 'document-capture-form', data: { liveness: liveness_checking_enabled?.presence } %>

```

Reply:
> WDYT of the rails tag helpers for this so we don't have to mangle strings? A `nil` value from (`prescence` converts all falsy values to nil) will be omitted in the rendered HTML
> 
> ```
> <%= tag.div id: 'document-capture-form', data: { liveness: liveness_checking_enabled?.presence } %>
> ```

I wasn't aware of this! But it looks much nicer. I'll update first thing on Monday.

Reply:
> WDYT of the rails tag helpers for this so we don't have to mangle strings? A `nil` value from (`prescence` converts all falsy values to nil) will be omitted in the rendered HTML
> 
> ```
> <%= tag.div id: 'document-capture-form', data: { liveness: liveness_checking_enabled?.presence } %>
> ```

Updated in aduth-split-locale-js.